arch/sim/Kconfig

```diff
@@ -531,16 +531,31 @@ config SIM_QSPIFLASH_PAGESIZE
 		"wrap" causing the initial data sent to be overwritten.
 		This is consistent with standard SPI FLASH operation.
 
+choice
```

Comment:
I don't think this should be choice as the options are not exclusive.   I don't see anything in the driver that would prevent you from exposing HCI0 as a tty device and HCI1 to the NuttX network stack.

Reply:
Yes, I will remove the choice option